api-scanner/src/test/resources/kotlin-internal-annotation/kotlin/net/corda/kotlin/AnnotatedClass.kt

```diff
@@ -0,0 +1,4 @@
+package net.corda.kotlin
```

Comment:
`net.corda.apiscanner.kotlin`

Reply:
These are test packages in distinct test Gradle projects. All of the rest are using `net.corda.example` packages except that IntelliJ is trying to conflate all of the classes into a single project. Perhaps `net.corda.example.kotlin` would be a better choice?